lib/livebook/runtime/evaluator.ex

```diff
@@ -873,11 +1023,18 @@ defmodule Livebook.Runtime.Evaluator do
           do: {:module, module},
           into: identifiers_used
 
+    # Erlang modules do not have the meta __info__/1 - thus we need to get it the old way.
     identifiers_defined =
-      for {module, _line_vars} <- tracer_info.modules_defined,
-          version = module.__info__(:md5),
-          do: {{:module, module}, version},
-          into: identifiers_defined
+      for {module, _line_vars} <- tracer_info.modules_defined, into: identifiers_defined do
+        {_type, version} =
+          case :erlang.function_exported(module, :__info__, 1) do
+            false -> {:erlang, module.module_info(:md5)}
+            true -> {:elixir, module.__info__(:md5)}
+          end
+
+        # IO.inspect("#{module} type: #{type} md5: #{version}")
+        {{:module, module}, version}
```

Comment:
The erlang version should work on both!

```suggestion
        {{:module, module}, module.module_info(:md5)}
```

Reply:
I will make sure - i had a test where a normal Elixir-module was not recognized.